slave-dotnet20/Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM openshift3/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet20-rhel7-docker" \
+      name="openshift3/jenkins-slave-dotnet20-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="4" \
+      io.k8s.display-name="Jenkins Slave .NET Core 2.0" \
+      io.k8s.description="The jenkins slave dotnet image has the dotnet tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,slave,dotnet,dotnet20"
+
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+# Install .NET Core
+RUN yum-config-manager --enable rhel-7-server-dotnet-rpms && \
+    yum-config-manager --disable epel >/dev/null || : && \
+    INSTALL_PKGS="rh-dotnet20" && \
```

Comment:
I assume we won't create a 1.x slave and start at 2.0 (due in a couple of months).